### PR TITLE
Add spreadsheet v2 and pod documentation

### LINKED ISSUE #86: Add definition of Activity Risk to spreadsheet
image

### LINKED ISSUE #210: [Spreadsheet] Activity risk > 100%
image

How about capping the activity risk at the "Partner" level?

### LINKED ISSUE #232: Spreadsheet: Add visual (emoji) key to spreadsheet to compare types of activities at a glance
image

I modified the spreadsheet to automatically generate an emoji-based key to help remind me if it was indoors/outdoors, masked/not, the distance, and the time. 

I find it easier to parse at a glance. Because anytime I'm reading your key or even re-reading custom scenarios that I made, I have to scroll to the right to remember the "settings"

Here's the formula: 
```
=IFS(
J3="One-time", (IFS(
  N3="Indoor", "🏠", N3="Outdoor", "🌲")&
  IF(P3="No mask", "💨", "😷")&
  IFS(
    Q3="> 6ft most of the time", "↔️", 
    Q3="Not distancing", "☕", 
    Q3="Cuddling", "❣️", 
    Q3="> 10ft most of the time","🏈"
  )&
  " ("&IF(K3, K3&"h", )&IF(L3, L3&"m", )&")"
),
J3="Housemate/repeated","👋👋👋",
J3="Partner", "💋💋💋",
ISBLANK(J3), ""
)
```

(If anyone has a better emoji suggestion other than 🏈  for "10+ feet distance", please let me know)

Note: I intentionally only represented the other person's mask since it has a bigger effect on the score.

You could also consider doing something like this to add more visual cues to the app.

### LINKED ISSUE #260: Spreadsheet: Use copy-only link
You can use

https://docs.google.com/spreadsheets/d/1DYIJgjG3H5rwt52NT2TX_m429snmIU-jGw1a8ZODwGQ/copy

as the link for the spreadsheet (note /copy suffix), and it will force everyone to make a copy right away.

If you make that change, though, you'd probably want to update the instructions on the spreadsheet.

### LINKED ISSUE #261: Spreadsheet: Add "custom people" risk modeler tab
I'm intending to add a way to model person risk in the spreadsheet.

I wanted to open an issue for it so I can get feedback on it as I go.

Here's a (non-functional) mockup of what I'm envisioning:

image

### LINKED ISSUE #321: Spreadsheet: Overhaul colors
Follow a color scheme similar to the one for our houses sheet. I haven't landed in the exact color set yet. My housemate is a designer and helping me narrow it down a bit to something more consistent and pleasant. The aim with the colors and fonts is to make the whole thing feel more gentle and accessible.

image
image

### LINKED ISSUE #322: Spreadsheet: Pod Overview graph and table
Here's the draft:

image

### LINKED ISSUE #323: Spreadsheet: Create documentation for pods / group houses
Create documentation about pods and group houses, and how they can best implement the spreadsheet>

I'm starting a working draft of this documentation in Google Docs here (anyone can leave a comment/suggestion): https://docs.google.com/document/d/1TfbcWjxr4kkzGhtGxcX9I4iJuVXrPFitWbTW6iDBzUU/edit#

(This blocks #322)

### LINKED ISSUE #332: Spreadsheet: Replace "cuddling" with "close" and "kissing"

### LINKED ISSUE #423: Spreadsheet: Add budget forecaster
image

### LINKED ISSUE #477: Create a way for spreadsheet users to subscribe to get updates about spreadsheet changes
Something with automatic email updates would be ideal, I think.

### LINKED ISSUE #478: Add a link to the spreadsheet changelog to the spreadsheet page on the site

### LINKED ISSUE #494: Spreadsheet: add HEPA options
To match #492 and PR #493

### LINKED ISSUE #506: Remove "delay factor" from prevalence for previous week's activities
> Now that we've got automatic location data in the spreadsheet, I'm noticing the whole situation of "my location data got updated, and all the activities I entered in the past week are now way higher" is a bigger problem. I'm thinking through possible solutions, and one of them effects the model, so I'd be curious your thoughts.
Obviously one solution is "copy-paste the row without formulas, then it's exactly the same as it was" is the most accurate. But it's also labor-intensive and requires some spreadsheet skills.
The alternate solution I'm considering right now is: what if we didn't apply the "delay factor" to any line items created before the "location data imported" date?
Example: Alameda County is 0.56% adjusted prevalence right now. The delay factor is 1.55. The data was generated on Nov 11. Is it legit to say that any activity items on Nov 11 or prior could count as actually 0.56%/1.55 = 0.37%?
Obviously, lines prior to the last week are not going to be accurate. But this would at least keep ones in the last 9 days more accurate.  Thoughts?

I already have the format written on my copy of the spreadsheet.

@beshaya said it makes sense

### PR DESCRIPTION
**Pull request by blanchardjeremy**
This pull request will contain the implementation of the "spreadsheet v2" documentation and pod/household documentation we've been working on in this [google doc](https://docs.google.com/document/d/1TfbcWjxr4kkzGhtGxcX9I4iJuVXrPFitWbTW6iDBzUU/edit#).

I'll have it as a draft pull request until it's ready for review.

**Major edits/additions can be found here:**

* https://deploy-preview-626--microcov.netlify.app/spreadsheet
* https://deploy-preview-626--microcov.netlify.app/spreadsheet/quickstart
* https://deploy-preview-626--microcov.netlify.app/spreadsheet/basics
* https://deploy-preview-626--microcov.netlify.app/spreadsheet/faq
* https://deploy-preview-626--microcov.netlify.app/spreadsheet/household-pod
* https://deploy-preview-626--microcov.netlify.app/paper/13-q-and-a#how-should-i-choose-my-annual-risk-budget -- I moved all the budget stuff from the [Riskiness page of the White Paper](https://deploy-preview-626--microcov.netlify.app/paper/2-riskiness) to this Q&A answer because I wanted it to all be in one place. I'm not convinced this is the best approach and would love feedback here. The other option I see is to move it all to the riskiness page. I just want spreadsheet users to be able to look it up easily.
* https://deploy-preview-626--microcov.netlify.app/symptoms
* https://deploy-preview-626--microcov.netlify.app/questions

**Closes the following issues:**
- Closes #86 (because this column is now hidden)
- Closes #210 
- Closes #232
- Closes #260 
- Closes #261
- Closes #321
- Closes #322
- Closes #323
- Closes #332
- Closes #423
- Closes #477
- Closes #478
- Closes #494
- Closes #506

### CODE REVIEW

**Comment by blanchardjeremy** [comment]
@linesarefuzzy when I am ready for a review, I'd love it if you were one of the folks who gave the final version of the docs a read-through. If you have time. :)

**Review by beshaya — CHANGES_REQUESTED** [review]
I'm just reviewing the code + paper changes right now, not the markdown contents of the spreadsheet docs. I'm happy submitting this to the feature branch once requested changes are made.

File: src/posts/risk-questions.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+const title = 'Risk Questions'
+const shortTitle = 'Risk Questions'
+const content = `
+The more you know about a person’s specific risks in the last 9 days, the better you can estimate the risk of being in contact with them.
+
+The **Option A** questions rae simpler, but don't provide as many prompts to assist you. The **Option B** question are more detailed to help you think through various kinds of possible activities.
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/posts/risk-questions.ts:6]: ✅

File: src/posts/paper/13-q-and-a.ts [hunk 2]
```diff
@@ -216,6 +268,8 @@ In terms of qualitative scales, we like the following:
 
 
 
+[^risktolerance]:
+     You can check the annual rate of COVID infection for your region by looking up the number of cases in the past _day_; for states in the US, try [covidactnow.org](http://covidactnow.org). Divide by your region’s population (or by 100,000 if the figure you got was “cases per 100,000 population”) and multiply by 365 (days per year) to get the proportion of your region that will have a reported case of COVID this year at current rates. You probably want to aim to stay below that percentage. For the San Francisco Bay Area in August 2020, it’s about 3.5%. There’s no correction for potential underreporting here, since we want to err on the _low_ side (not take too many risks).
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/posts/paper/13-q-and-a.ts:272]: nuked

File: src/pages/Symptoms.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { MarkdownContents } from 'components/markdown/PaperPage'
+import { PostMap } from 'posts/post'
+import symptomsPage from 'posts/symptoms'
+
+// Note: the keys in this map are the public URL for the post. Changing them will break links.
+const symptomPostMap: PostMap = {
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/pages/Symptoms.tsx]: Sure I can see the case for that.

[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/pages/Symptoms.tsx]: Done.

File: src/pages/Spreadsheet.tsx [hunk 4]
```diff
@@ -24,79 +31,147 @@ export const SubscribeForm: React.FunctionComponent = () => {
...
-          Sign up for spreadsheet updates
-        </a>
-      </p>
+      <p>{mailchimpLink()}</p>
     </>
   )
 }
 
-export const Spreadsheet = (): React.ReactElement => {
+export const SpreadsheetTOC = (): React.ReactElement => {
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/pages/Spreadsheet.tsx:39]: Can you say more about what you mean? This is following the same pattern as `Paper.tsx` and `PaperTOC`

What different file would you like it refactored into?

File: src/components/markdown/PaperPage.tsx [hunk 5]
```diff
@@ -11,7 +11,19 @@ export const MarkdownContents: React.FunctionComponent<{
   posts: PostMap
   id: string
   allInOnePage?: boolean
-}> = ({ posts, id, allInOnePage }) => {
+  rootTitle?: string
```
[the remark this answers is not in the document]
Comment (blanchardjeremy) [reply src/components/markdown/PaperPage.tsx:14]: So after reviewing this, I see that it makes sense to remove `simplyLayout`. I don't see anything else that I would remove. 

If I refactored this into a separate file, I would actually keep `secitonTitle` as a feature. I think A is more clear than B:
image
image

If I went with your approach of separating the files, I would
* Hardcode `sectionTitle` into the body of the function
* Remove `rootTitle` and put it back in hard-coded as "Table of Contents" in the body of the function
* Remove `showSectionNumbers`

These are pretty minor additions IMO. 

Whereas if I keep it as is, I don't have to replicate the entire body of the function and keep it up to date if we ever improve our page layout.

So... Those are my thoughts. If you still feel strongly about it, I'll just copy-paste it into its own file with these changes. Just seems odd to me.

**Comment by beshaya** [comment]
That approach seems reasonable to me.

**Comment by beshaya** [comment]
I think that pattern isn't good, but I can clean it up later if you want to
keep it consistent in this PR.

**Pull request merged**